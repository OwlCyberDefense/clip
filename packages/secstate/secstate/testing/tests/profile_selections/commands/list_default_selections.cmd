/usr/sbin/secstate list -r -a
