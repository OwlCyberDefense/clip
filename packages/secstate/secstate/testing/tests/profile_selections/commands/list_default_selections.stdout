[X]Benchmark - ID: PassComp, Title: 'Password Complexity', Profile: None
	[ ]profile - id: minlen-deselected, title: 'minimum length rule deselected'
	[ ]profile - id: passgroup-deselected, title: 'password group deselected'
	[ ]profile - id: minlen-long, title: 'long minimum length'
	[ ]profile - id: minlen-short, title: 'short minimum length'
	[ ]profile - id: minlen-set, title: 'custom minimum length'
	[x]profile - id: none, title: 'none'
	[X]Group - ID: PassComp-G-2-2, Title: 'Password'
		[X]Group - ID: PassComp-G-2-3, Title: 'Password Complexity'
			[X]Rule - ID: PassComp-R-2-1, Title: 'Lowercase'
			[X]Rule - ID: PassComp-R-2-2, Title: 'Min. Length'
			[X]Rule - ID: PassComp-R-2-3, Title: 'Numeric'
			[X]Rule - ID: PassComp-R-2-4, Title: 'Special'
			[X]Rule - ID: PassComp-R-2-5, Title: 'Uppercase'
